README.md

```diff
@@ -17,3 +17,14 @@ The entire content of this repository should be regarded as highly
 unstable until this warning is removed. The schema may change, the
 CloudEvent "type" attribute values may change, the protobuf message
 may change. Even the core `cloud_event_type` annotation may change.
+
+# CloudEvent Types
+
+The following list is generated from the contents in the `proto/` directory.
+
+- [Google.Events.Protobuf.Firebase.V1](./proto/google/events/firebase/v1/data.proto)
```

Comment:
These names are the C# names. The message names themselves are in protobuf packages called `google.events.firebase.v1` etc.
I think we should keep language-neutral for this.

Reply:
Fixed.